akka-http/src/main/scala/akka/http/javadsl/server/RejectionHandler.scala

```diff
@@ -173,6 +173,12 @@ abstract class RejectionHandler {
   def handleValidationRejection(ctx: RequestContext, message: String, cause: Throwable): RouteResult = passRejection()
 
   /**
+   * Callback called to handle rejection created by the onCompleteWithBreaker directive.
+   * Signals that the request was rejected because the supplied circuit breaker is open and requests are failing fast.
+   */
+  def handleCircuitBreakerOpenRejection(ctx: RequestContext): RouteResult = passRejection()
```

Comment:
thanks a lot for doing java side of it too! (this will be removed once we merge new javadsl)